v0/destinations/ga4/networkHandler.js

```diff
@@ -0,0 +1,92 @@
+const {
+  proxyRequest,
+  prepareProxyRequest
+} = require("../../../adapters/network");
+const {
+  getDynamicMeta,
+  processAxiosResponse
+} = require("../../../adapters/utils/networkUtils");
+const {
+  isDefinedAndNotNull,
+  isDefined,
+  isHttpStatusSuccess
+} = require("../../util");
+const { TRANSFORMER_METRIC } = require("../../util/constant");
+const ErrorBuilder = require("../../util/error");
+
+const responseHandler = (destinationResponse, dest) => {
+  const message = `[GA4 Response Handler] - Request Processed Successfully`;
+  let { status } = destinationResponse;
+  const { response } = destinationResponse;
+  if (status === 204) {
+    // GA4 always returns a 204 response, other than in case of
+    // validation endpoint.
+    status = 200;
+  } else if (
```

Comment:
since we are converting 204 -> 200; should we make else if to independent if block